`parallel/test-util-isDeepStrictEqual.js` regression

Test update to Node.js 24.12.0 in https://github.com/denoland/deno/pull/31628 causes `parallel/test-util-isDeepStrictEqual.js` regression:
```
error: AssertionError: Test "parallel/test-util-isDeepStrictEqual.js" failed: running 2 tests from ./test/parallel/test-util-isDeepStrictEqual.js
util.isDeepStrictEqual with skipPrototype ... FAILED (4ms)
util.isDeepStrictEqual skipPrototype with complex scenarios ... FAILED (0ms)

 ERRORS 

util.isDeepStrictEqual with skipPrototype => ./test/parallel/test-util-isDeepStrictEqual.js:99:3
error: AssertionError: Expected values to be strictly equal:

false !== true

    at new AssertionError (ext:deno_node/assertion_error.ts:531:11)
    at toNode (node:assert:98:15)
    at assert.strictEqual (node:assert:345:3)
    at file:///home/admin/actions-runner/_work/deno/deno/tests/node_compat/runner/suite/test/parallel/test-util-isDeepStrictEqual.js:108:12
    at node:test:267:13

util.isDeepStrictEqual skipPrototype with complex scenarios => ./test/parallel/test-util-isDeepStrictEqual.js:126:3
error: AssertionError: Expected values to be strictly equal:

false !== true

    at new AssertionError (ext:deno_node/assertion_error.ts:531:11)
    at toNode (node:assert:98:15)
    at assert.strictEqual (node:assert:345:3)
    at file:///home/admin/actions-runner/_work/deno/deno/tests/node_compat/runner/suite/test/parallel/test-util-isDeepStrictEqual.js:138:12
    at node:test:267:13

 FAILURES 

util.isDeepStrictEqual with skipPrototype => ./test/parallel/test-util-isDeepStrictEqual.js:99:3
util.isDeepStrictEqual skipPrototype with complex scenarios => ./test/parallel/test-util-isDeepStrictEqual.js:126:3
```

--- a/tests/node_compat/config.jsonc
+++ b/tests/node_compat/config.jsonc
@@ -1413,12 +1413,7 @@
     "parallel/test-util-deprecate-invalid-code.js": {},
     "parallel/test-util-inherits.js": {},
     "parallel/test-util-inspect-getters-accessing-this.js": {},
-    "parallel/test-util-isDeepStrictEqual.js": {
-      "darwin": false,
-      "linux": false,
-      "windows": false,
-      "reason": "https://github.com/denoland/deno/issues/31632"
-    },
+    "parallel/test-util-isDeepStrictEqual.js": {},
     "parallel/test-util-primordial-monkeypatching.js": {},
     "parallel/test-util-text-decoder.js": {},
     "parallel/test-util-types-exists.js": {},

